wp-content/themes/engage/assets/scss/components/_widget.scss

```diff
@@ -29,34 +29,59 @@
 		color: $black-grey;
 	}
 
-	.constantcontactwidget_form {
-		p {
-			margin-bottom: 1rem;
-			color: $black-grey;
-		}
+	.textwidget {
```

Comment:
Instead of hooking into `.textwidget` can you add a custom css class to the same level as `.wpcf7-form` and hook into that? In general we want our CSS to have as low a specificity as possible, so the fewer classes/attributes in the chain the better. That keeps things easy to override.

Reply:
How about not using either of those. I got it to work without needing to hook into either of them.

Reply:
Even better :)